CONTRIBUTING.md

```diff
@@ -21,6 +21,22 @@ To install in development mode, follow these instructions.
 -   Test that your installation was succesful by running `esmvaltool -h`.
 -   If you log into a cluster or other device via `ssh` and your origin machine sends the `locale` environment via the `ssh` connection, make sure the environment is set correctly, specifically `LANG` and `LC_ALL` are set correctly (for GB English UTF-8 encoding these variables must be set to `en_GB.UTF-8`; you can set them by adding `export LANG=en_GB.UTF-8` and `export LC_ALL=en_GB.UTF-8` in your origin or login machines' `.profile`)
 
+## Getting started on MacOS OSX-64
+To install on a MacOS you will have to perform the following installation procedure:
+
+**WARNING**
+The Mac OSX installation is not being continuously tested via CircleCI! If you notice any problems please report them on GitHub.
+
+  - Download and install Anaconda [for OSX](https://repo.anaconda.com/archive/Anaconda3-2019.07-MacOSX-x86_64.sh)(install by e.g. `bash Anaconda3-2019.07-MacOSX-x86_64.sh` or whatever the installer is called);
+  - Answer yes when, at the end of the installation, Anaconda asks you to initialize the installation in your `.bash_profile`;
+  - Clone the ESMValTool Core Git repository: `git clone git@github.com:ESMValGroup/ESMValCore`
+  - Go to the source code directory: `cd ESMValCore`
+  - Use the `environment.yml` file to create the conda viartual environment, as explained above for a Linux installation: `conda env create -n esmvaltool -f environment.yml`
+  - Activate the esmvaltool environment: `conda activate esmvaltool`
+  - Install in development mode: `pip install -e '.[develop]'`. If you are installing behind a proxy that does not trust the usual pip-urls you can declare them with the option `--trusted-host`, e.g. `pip install --trusted-host=pypi.python.org --trusted-host=pypi.org --trusted-host=files.pythonhosted.org -e .[develop]`
+  - Test that your installation was succesful by running `esmvaltool -h`.
+  - Note that the installation instructions are identical to the Linux installation due to the package being architecture agnostic (specified by the `noarch` label).
```

Comment:
Is this note really useful? After all, we just read through a separate set of instructions and at any rate, who will care?

Reply:
yeah good point dude, I changed it